ethcore/src/trace/types/trace.rs

```diff
@@ -141,13 +141,19 @@ pub enum RewardType {
 	Block,
 	/// Uncle
 	Uncle,
+	/// Empty step (AuthorityRound)
```

Comment:
Should we remove this type from tracing and instead use `block_reward::RewardKind`?

Reply:
Sounds ok for me.